hoomd/operations.py

```diff
@@ -117,29 +213,76 @@ def integrator(self, op):
 
     @property
     def updaters(self):
+        """list[`hoomd.operation.Updater`]: A list of updater operations.
+
+        Holds the list of updaters associated with this collection. The list can
+        be modified as a standard Python list.
+        """
         return self._updaters
 
     @property
     def analyzers(self):
+        """list[`hoomd.operation.Analzyer`]: A list of analyzer operations.
+
+        Holds the list of analyzers associated with this collection. The list
+        can be modified as a standard Python list.
+        """
         return self._analyzers
 
     @property
     def tuners(self):
+        """list[`hoomd.operation.Tuner`]: A list of tuner operations.
+
+        Holds the list of tuners associated with this collection. The list can be
+        modified as a standard Python list.
+        """
         return self._tuners
 
     def __iadd__(self, operation):
+        """Works the same as `Operations.add`.
+
+        Args:
+            operation (``operation``): A HOOMD-blue updater, analyzers, compute,
+                tuner, or integrator to add to the object.
+        """
         self.add(operation)
+        return self
 
     def remove(self, operation):
+        """Remove an operation from the `Operations` object.
+
+        Remove the item from the collection whose id is the same as
+        ``operation``. See
+        `<https://docs.python.org/3/library/functions.html#id>`_ for the concept
+        of a Python object id.
+
+        Args:
+            operation (``operation``): A HOOMD-blue integrator, updater,
+                analyzer, tuner, or compute, to remove from the container.
+
+        Raises:
+            ValueError: raises if operation is not found in this container.
+            TypeError: raises if operation is not of a valid type for this
+                container.
```

Comment:
I'm adopting the "Raises" phrasing structure from [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) in this suggestion. I prefer its conciseness.
```suggestion
            ValueError: If ``operation`` is not found in this container.
            TypeError: If ``operation`` is not of a valid type.
```